tasks/cycles/cycles.js

```diff
@@ -42,6 +113,13 @@ function calculateDiscount(redemption) {
     interestOnDeposit: 250,
     otherExpences: -300
   };
+  var totalIncome = 0;
+  var x;
+  for (x in personIncomes) {
```

Comment:
define the variable inside the loop
for ( let x in personIncomes )

Reply:
Done